### DISCUSSION TITLE
Der TLA sollte ein Login-Funktion haben, über die man Zugriff auf zusätzliches Material hat

### DISCUSSION THREAD

**Discussion by LutzPopko** [thesaurus-linguae-aegyptiae/tla-web#117] [Ideas]
Der alten TLA enthält Textdaten, die wir im neuen TLA wegen fehlender Genehmigung der Urheber nicht nutzen dürfen (Stichwort: Edfu-Texte). Sobald der alte TLA komplett abgeschaltet wird, wären diese Texte verloren. Daher wäre es überlegenswert, sie auch im neuen TLA zugänglich zu machen, aber nur so, dass man sich vorher einloggen muss. Auf diese Weise kann der Zugriff bspw. auf Projektmitarbeiter o.ä. beschränkt werden.

**Comment by schnoepf** [comment]
ich könnte versuchen, das alte TLA webzuarchivieren. So wie unter webarchive.bbaw.de schon einige Ressourcen zu finden sind.

**Comment by dwerning** [comment]
Aktuell ist ein Login aufgrund der DSVGO-Komplikationen im neuen TLA nicht geplant. Auch wurde von einem Beiratsmitglied von der Mischung aus CC-BY und unfreien Daten abgeraten.
Der **lagacyTLA soll also aus verschiedenen Gründen so lange wie er durchhält online bleiben** (auch werden wir die volle Funktionalität des lagacyTLA nicht zeitnah im neuen TLA erreichen).
Ich transformiere diesen Issue in eine Diskussion. (Achtung der Repro ist weltweit einsehbar; bitte nicht als Chat benutzen.)
